### DISCUSSION TITLE
Mapping string with numbers

### DISCUSSION THREAD

**Discussion by carfermar** [JoshClose/CsvHelper#1543] [Q&A]
I have following classes

public class A
{
   string Code {get; set}
}

public ACsvMap()
{
  AutoMap();
}

I receive from csv following values: 
0001  
001
01
201806190060

Execution returns 
1
1
1
2E+11

I cannot find any solution to solve this problem.

**Comment by zeug-it** [comment]
this is similar to the removed + char. if you have a string 2019/11 + 2019/10 in a cell the plus char is removed in the imported result even if the field is a string.
